typegen/src/utils.rs

```diff
@@ -263,7 +294,7 @@ mod generics_list {
     struct GenericsListInner {
         previous: Option<GenericsList>,
         start_idx: usize,
-        generics_by_id: Vec<u32>,
+        generics_by_id: Vec<(Option<u32>, usize, String)>,
```

Comment:
Perosnally I'd remove the `usize` index from here; it's redundant information since it's just the index inthe vec. I'd prefer to just add `.enumerate()` where we want this.

Reply:
removed it